src/content/translations/fr/contributing/translation-program/translators-guide/index.md

```diff
@@ -2,6 +2,7 @@
 title: Guide à l'intention des traducteurs
 lang: fr
 description: Instructions et conseils pour les traducteurs d'ethereum.org
+sidebar: true
```

Comment:
@minimalsm These should be removed, correct?

Reply:
Yup, thanks for catching.